script.js

```diff
@@ -0,0 +1,67 @@
+const movie = [
+    {
+    title: "The Shawshank Redemption",
+  },
+  {
+    title: "The Godfather",
+  },
+  {title: "The Dark Knight",},
+  {
+    title: "12 Angry Men",
+  },
+  {
+    title: "The Lord Of The Rings",
+  },
+  {
+    title: "Pulp Fiction",
+  },
+  {
+    title: "The Good, the Bad and the Ugly",
+  },
+  {
+    title: "Forrest Gump",
+  }
+]
```

Comment:
It is a good forward thinking idea to wrap the titles into objects, since it might be that in the future, you will add the length of each movie, or its IMDb rating.